### PR TITLE
Minimal refactoring

### PR DESCRIPTION
**Pull request by aplanas**
This does not change any semantic, but is a pain to rebase.

### CODE REVIEW

File: git-importer.py [hunk 1]
```diff
@@ -8,14 +8,14 @@
 import pathlib
 import sys
 import time
+import xml.etree.ElementTree as ET
```
Comment (coolo) [inline git-importer.py:11]: Sorry no. all of openSUSE-release-tools is using lxml. that's the standard for osrt

Comment (aplanas) [reply git-importer.py:11]: If you want some speed, osclib / osc / requests should be dropped too.

But in any case this change affects the importer code only. The usage of lxml or not here is orthogonal of what osc / osclib uses internally. The importer works the same.

Comment (coolo) [reply git-importer.py:11]: Then let's agree that this will stay in my branch - or even move into a repo on its own.

Comment (aplanas) [reply git-importer.py:11]: I can even agree on revert this change on Monday.  There is more value on working on the same codebase that selecting one parser or another.

But I still think that I was not able to explain why is irrelevant the change.

The importer uses the lxml parser to convert a file descriptor into an object that can be walked, and we walk it inside the importer.

The change use the standard parser to convert a file descriptor into an object that can be walked, ... and we that do again in the importer, with the same API.

It is irrelevant the parser that osc uses for that.  We can even use a regular expression (sorry) or something else. Osc GET() gives us a file descriptor, we can do what ever we want with it as we never pass the parser back to the osc library.

Comment (coolo) [reply git-importer.py:11]: So it broke alsa import.

**Pull request merged**